### PR TITLE
specify enabling redis module

### LINKED ISSUE #2517: Redis as a Caching Backend Doc Update
Re: [Redis as a Caching Backend](https://pantheon.io/docs/redis/)

Priority (Low‚ Medium‚ High): Low

## Issue Description:

Tried installing Redis for the most recent version of D8 by following the docs instructions - not working. Error below.

```
[30-May-2017 08:24:21 UTC] Symfony\Component\DependencyInjection\Exception\LogicException: Service 'cache_tags.invalidator.checksum' for consumer 'cache_tags.invalidator' does not implement Drupal\Core\Cache\CacheTagsInvalidatorInterface. in /srv/bindings/28c6db6d1d9045428ea4ba8255c7dda6/code/core/lib/Drupal/Core/DependencyInjection/Compiler/TaggedHandlersPass.php on line 126 #0 /srv/bindings/28c6db6d1d9045428ea4ba8255c7dda6/code/vendor/symfony/dependency-injection/Compiler/Compiler.php(104): Drupal\Core\DependencyInjection\Compiler\TaggedHandlersPass->process(Object(Drupal\Core\DependencyInjection\ContainerBuilder))
```

## Suggested Resolution

### PR DESCRIPTION
**Pull request by alexfornuto**
Closes #2517 

## Effect
PR includes the following changes:
- Specifies need to activate Redis module before continuing.

## Remaining Work
- [x] 👍  from @alexdicianu 
- [x] Copy Review

### CODE REVIEW

File: source/_docs/redis.md [hunk 1]
```diff
@@ -36,7 +36,11 @@ For detailed information, see [Installing Redis on WordPress](/docs/wordpress-re
 
 1. Enable the Redis cache server from your Pantheon Site Dashboard by going to **Settings** > **Add Ons** > **Add**.
 
-2. Add the [Redis](https://www.drupal.org/project/redis) module from Drupal.org.
+2. Install and activate the [Redis](https://www.drupal.org/project/redis) module from Drupal.org.
+
+    <div class="alert alert-info">
+    <h4 class="info">Note</h4><p markdown="1">You **Must** activate the plugin before proceeding.
```
Comment (alexdicianu) [inline source/_docs/redis.md]: Drupal people call these modules :) There will be confusion if you call them plugins, as Drupal 8 also has plugins that do differnt things.

File: source/_docs/redis.md [hunk 2]
```diff
@@ -36,8 +36,12 @@ For detailed information, see [Installing Redis on WordPress](/docs/wordpress-re
 
 1. Enable the Redis cache server from your Pantheon Site Dashboard by going to **Settings** > **Add Ons** > **Add**.
 
-2. Add the [Redis](https://www.drupal.org/project/redis) module from Drupal.org.
-
+2. Install and activate the [Redis](https://www.drupal.org/project/redis) module from Drupal.org.
```
Comment (rachelwhitton) [inline source/_docs/redis.md]: Suggest adding the Terminus command to install and enable the module for D8 and D7 while we're here

**Comment by rachelwhitton** [comment]
+1

File: source/_docs/redis.md (hunk 2 above)
Comment (alexfornuto) [inline source/_docs/redis.md]: Good idea! Completed on L45-57 and L80-82

**Pull request merged**